packages/core/src/transport/eventBridge.spec.ts

```diff
@@ -33,3 +33,28 @@ describe('isEventBridgePresent', () => {
     })
   })
 })
+
+describe('getEventBridge', () => {
+  let eventBridgeStub: DatadogEventBridge
+
+  beforeEach(() => {
+    updateExperimentalFeatures(['event-bridge'])
+    eventBridgeStub = initDatadogEventBridgeStub()
+  })
+
+  afterEach(() => {
+    resetExperimentalFeatures()
+    deleteDatadogEventBridgeStub()
+  })
+
+  it('should serialize the event for the datadog bridge', () => {
+    const sendSpy = spyOn(eventBridgeStub, 'send')
+    const bridge = getEventBridge()
+    const event = { foo: 'bar' }
+    const eventType = 'view'
+
+    bridge.send(eventType, event)
+
+    expect(sendSpy).toHaveBeenCalledOnceWith(JSON.stringify({ eventType, event }))
```

Comment:
what about expecting an actual string value instead of rewriting the implementation in the test?